hnn_core/network.py

```diff
@@ -223,6 +224,9 @@ def __init__(self, params, add_drives_from_params=False,
         self.external_drives = dict()
         self.external_biases = dict()
 
+        # network connectivity
+        self.connectivity_list = list()
```

Comment:
```suggestion
        self.connectivity= list()
```

user can just do `type(connectivity)` to see object. Also you should document it above